### PR TITLE
Integrated DBR reshard list and cancel command in suite

### PR DESCRIPTION
**Pull request by udaysk23**
# Description
Integrated DBR reshard list and concel command in suite
Following is the log link:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-XD6FFE

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/pacific/rgw/tier-1-extn_rgw_multisite-secondary-to-primary.yaml [hunk 1]
```diff
@@ -269,3 +269,15 @@ tests:
             config-file-name:  test_datalog_trim_command.yaml
             timeout: 300
             verify-io-on-site: ["ceph-pri"]
+  - test:
+      name: Test DBR reshard list and concel command on secondary
```
Comment (anrao19) [inline suites/pacific/rgw/tier-1-extn_rgw_multisite-secondary-to-primary.yaml:273]: Please correct the typo

File: suites/pacific/rgw/tier-1_rgw_multisite_test-upgrade-4-to-latest.yaml [hunk 2]
```diff
@@ -402,3 +402,15 @@ tests:
       module: sanity_rgw_multisite.py
       name: Manual Resharding tests on Primary cluster
       polarion-id: CEPH-83574734
+  - test:
+      name: Test DBR reshard list and concel command on primary
+      desc: Test DBR reshard list and concel command on primary
```
Comment (anrao19) [inline suites/pacific/rgw/tier-1_rgw_multisite_test-upgrade-4-to-latest.yaml:407]: Please correct the typo

**Review by anrao19 — APPROVED** [review]

**Review by Ameenasuhani — COMMENTED** [review]
@udaysk23 Please squash your commits

**Review by ckulal — DISMISSED** [review]

**Review by Ameenasuhani — APPROVED** [review]
LGTM

**Review by anrao19 — APPROVED** [review]

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**